config_scheduler.go

```diff
@@ -18,6 +18,7 @@ type SchedulerConfig struct {
 	TargetTimeSeconds             int     `bson:"target_time_seconds" json:"target_time_seconds" mapstructure:"target_time_seconds"`
 	AcceptableHostIdleTimeSeconds int     `bson:"acceptable_host_idle_time_seconds" json:"acceptable_host_idle_time_seconds" mapstructure:"acceptable_host_idle_time_seconds"`
 	GroupVersions                 bool    `bson:"group_versions" json:"group_versions" mapstructure:"group_versions"`
+	RoundRule                     int     `bson:"round_rule" json:"round_rule" yaml:"round_rule"`
```

Comment:
I'm not sure actually if it's still used, but you should include the mapstructure tag that the other variables have, to be safe (and it doesn't look like yaml is necessary).